features/adults/conviction_custodial_sentence.feature

```diff
@@ -100,6 +100,38 @@ Feature: Conviction
     And I click the "Continue" button
     Then I should see "Were any of the offences specified in Schedule 18 of the sentencing code?"
 
+    When I choose "Yes"
+    Then I should see "Was more than one sentence given at the same time?"
+
     When I choose "Yes"
     And I check my "conviction" answers and go to the results page
     Then I should see "This conviction will be spent on 31 December 2031"
+
+@happy_path @date_travel
```

Comment:
I think this scenario is in the wrong place as this feature covers adult convictions.